PreFreeSurfer/scripts/T2wToT1wReg.sh

```diff
@@ -37,14 +37,23 @@ OutputT2wTransform="${10}"
 T1wImageBrainFile=`basename "$T1wImageBrain"`
 
 cp "$T1wImageBrain".nii.gz "$WD"/"$T1wImageBrainFile".nii.gz
-${FSLDIR}/bin/epi_reg --epi="$T2wImageBrain" --t1="$T1wImage" --t1brain="$WD"/"$T1wImageBrainFile" --out="$WD"/T2w2T1w
-${FSLDIR}/bin/applywarp --rel --interp=spline --in="$T2wImage" --ref="$T1wImage" --premat="$WD"/T2w2T1w.mat --out="$WD"/T2w2T1w
-${FSLDIR}/bin/fslmaths "$WD"/T2w2T1w -add 1 "$WD"/T2w2T1w -odt float
+
+if [ "${T2wImage}" = "NONE" ] ; then
+    log_Msg "Skipping T2w to T1w registration --- no T2w image."
+else
+    ${FSLDIR}/bin/epi_reg --epi="$T2wImageBrain" --t1="$T1wImage" --t1brain="$WD"/"$T1wImageBrainFile" --out="$WD"/T2w2T1w
+    ${FSLDIR}/bin/applywarp --rel --interp=spline --in="$T2wImage" --ref="$T1wImage" --premat="$WD"/T2w2T1w.mat --out="$WD"/T2w2T1w
+    ${FSLDIR}/bin/fslmaths "$WD"/T2w2T1w -add 1 "$WD"/T2w2T1w -odt float
+fi
+
 cp "$T1wImage".nii.gz "$OutputT1wImage".nii.gz
 cp "$T1wImageBrain".nii.gz "$OutputT1wImageBrain".nii.gz
 ${FSLDIR}/bin/fslmerge -t $OutputT1wTransform "$T1wImage".nii.gz "$T1wImage".nii.gz "$T1wImage".nii.gz
 ${FSLDIR}/bin/fslmaths $OutputT1wTransform -mul 0 $OutputT1wTransform
-cp "$WD"/T2w2T1w.nii.gz "$OutputT2wImage".nii.gz
-${FSLDIR}/bin/convertwarp --relout --rel -r "$OutputT2wImage".nii.gz -w $OutputT1wTransform --postmat="$WD"/T2w2T1w.mat --out="$OutputT2wTransform"
+
+[ "${T2wImage}" != "NONE" ] && cp "$WD"/T2w2T1w.nii.gz "$OutputT2wImage".nii.gz
+[ "${T2wImage}" != "NONE" ] && ${FSLDIR}/bin/convertwarp --relout --rel -r "$OutputT2wImage".nii.gz -w $OutputT1wTransform --postmat="$WD"/T2w2T1w.mat --out="$OutputT2wTransform"
```

Comment:
For general readability of code, rather than using two lines that both use the `&&` construct here, I'd suggest using a "standard" `if` conditional.

Reply:
The impetus was to have minimal change that is clearly evident and avoid multitude if/else/fi conditionals.

Reply:
I am okay with the && usage

Reply:
Changed to the style prevalent in existing HCP code with if conditionals.